sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -148,7 +148,7 @@ private Analytics() {
      *
      * @return shared instance.
      */
-    @SuppressWarnings("WeakerAccess")
+    @SuppressWarnings({"WeakerAccess", "RedundantSuppression"})
```

Comment:
So we need this because we call it via reflection.
The batch code analysis will say it can be package-private so we suppress with "WeakerAccess", the in file edit code analysis will disagree with the batch analysis... so we need to add the second suppress to reconcile them...